recipes/nds2-client-swig/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "nds2-client-swig" %}
+{% set version = "0.16.0" %}
+{% set sha256 = "db785868f519a07e3a1cf7216cf5610a984d96e3795862794de98215534aa463" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.xz
+  url: http://software.ligo.org/lscsoft/source/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - make
+    - cmake >=3.6
+    - pkg-config
+    - swig >=3.0.7
```

Comment:
Isn't SWIG a host dependency?

Reply:
@scopatz, I honestly have no idea, but I moved it and the build didn't fail...